### DISCUSSION TITLE
Is anyone use freeverb stk??

### DISCUSSION THREAD

**Discussion by SandeepJagdev** [pschatzmann/arduino-audio-tools#1107] [Q&A]
Hello is anyone has reference code for freeverb stk effect??

**Comment by weow147** [comment]
hi sandeep - freeverb sounds terrible for me.  chowning reverb much better

**Reply by GFelipeSanchezS** [reply]
Which one is taht one?

**Comment by pschatzmann** [comment]
Just have a look at the [available class documentation](https://pschatzmann.github.io/arduino-audio-tools/classaudio__tools_1_1_audio_effect.html)
